config_mac.c

```diff
@@ -147,16 +153,21 @@ char *proxy_config_mac_get_bypass_list(void) {
         return NULL;
 
     // Get whether to exclude simple hostnames
-    bool exclude_simple_hostnames = get_cf_dictionary_bool(proxy_settings, kCFNetworkProxiesExcludeSimpleHostnames);
+    bool exclude_simple_hostnames = false;
```

Comment:
```suggestion
    bool exclude_simple_hostnames = false;
    CFArrayRef exceptions_list = NULL;
```

Reply:
I would prefer to keep the variables close to where they are being used and live with having two `#if/#endif`.

Reply:
I don't want to combine logic sections in order to merge two `#if/defs`.